### DISCUSSION TITLE
Motion detection after waking up the camera.

### DISCUSSION THREAD

**Discussion by Lahks** [gopro/labs#707] [Q&A]
Hello, 
I'm trying to configure Hero 9 (and possibly MAX in a future if that will work) for motion detection during specific hours, so it will not waste battery on frying sensor during the night, or fill the card with unusable footage. Motion detection itself works, and waking up camera to record at specific time works as well. But when I set it to wake up at particular time and then detect motion, it does not work: Wakes at the set time, displays "difference 0" for a brief time and stops the script completly, nothing else happens no matter if there is a motion in front of the lens or not.
Ideal would be a script that will wake up every day at specific time, detect motion, shut down after a couple of hours and repeat next day.
Maybe You have any ideas how to solve it, or at least make it wake up and detect? Also it would be lovely to keep the rear screen off until touched, because it's a biggest battery drainer, and requires taping to not scare the wildlife.
Regards, and thanks for a great job with the labs :)

**Comment by dnewman-gpsw** [comment]
You should share the scripts you are trying.  

This will be easier on MAX/HERO11 or 12 (best) as it they have a better/newer scripting engines.

This is only way to do on older cameras (10 or older)
<09:00!09:00N>18:00!09:00N!SM3D4H5!R

Test the time and sleep in out of range, otherwise motion detect.

**Reply by Lahks** [reply]
Thanks for Your answer!
Tested it, and tried the simplest versions such as !60N!SM3D4H5!R and !60SM3D4H5!R
And the result is the same i got before. Wakes properly, tries to start motion detection, displays "difference 0" for a short moment, but does not react to the movement, stops the script, and does nothing. :/
!SM3D4H5!R on its own works well.

**Reply by dnewman-gpsw** [reply]
I have confirmed your findings and there does seem to be a bug on HERO9. I tried several workarounds with no luck. I did test the same on HERO10 and 12, and the bug is not there. Unfortunately there are no more Labs releases planned for HERO9 (or any GP1 camera.)

**Reply by Lahks** [reply]
That's a bummer, but thanks a lot for checking it. 
By the way, could someone by any chance check if it works with MAX? I'm considering getting it specifically for this purpose.
Cheers!
